iota/commands/core/find_transactions.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
             f.Array
           | f.FilterRepeater(
                 f.Required
-              | Trytes(result_type=Address)
+              | AddressNoChecksum(result_type=Address)
```

Comment:
❔ Is it possible to design `AddressNoChecksum` so that its initialiser doesn't require a `result_type` argument?  Or at least, make it optional?

I'm trying to think of a scenario where a developer would want to use something other than `Address` there, but I can't think of anything.

Reply:
Fair, I think that will inherently clean up a bit too.